src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -339,18 +339,36 @@ private static void profileAccounting(ExecutedActionMetadata executedActionMetad
         executedActionMetadata.getWorkerStartTimestamp(),
         REMOTE_QUEUE,
         "queue");
+    logProfileTask(
+        converter,
+        executedActionMetadata.getWorkerStartTimestamp(),
+        executedActionMetadata.getInputFetchStartTimestamp(),
+        REMOTE_QUEUE,
+        "pre-fetch");
     logProfileTask(
         converter,
         executedActionMetadata.getInputFetchStartTimestamp(),
         executedActionMetadata.getInputFetchCompletedTimestamp(),
         REMOTE_SETUP,
         "fetch");
+    logProfileTask(
+        converter,
+        executedActionMetadata.getInputFetchCompletedTimestamp(),
+        executedActionMetadata.getExecutionStartTimestamp(),
+        REMOTE_QUEUE,
```

Comment:
I'm not sure how much it matters, but should this be `REMOTE_SETUP` to keep with the pattern?